shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitAnalysisResult.java

```diff
@@ -60,4 +45,9 @@ public ShortCircuitAnalysisResult setNetworkMetadata(NetworkMetadata networkMeta
         return this;
     }
 
+    public Map<String, List<LimitViolation>> getLimitViolations() {
```

Comment:
To remove, instead we could just get a FaultResult by its ID:

getFaultResult(String id)